extension-support/src/util/handlebars-utils.ts

```diff
@@ -33,7 +33,7 @@ export function compileTemplate(filename: string, variables: any): Promise<strin
     return new Promise((resolve, reject) => {
         fs.readFile(filename, 'utf-8', (error, source) => {
             // Register any helpers we need
-            const template = handlebars.compile(source);
+            const template = handlebars.compile(source, {noEscape: true});
```

Comment:
Wow I didn't know this parameter was a thing! I've just been using the triple-bracket method in places where the escaping could be a problem: `{{{value}}}`. I much prefer this though so we can just use the double-bracket syntax everywhere.